istio/1.12/patches/istio/20230603-unwatch-webhook.patch

```diff
@@ -0,0 +1,18 @@
+diff --color -Naur external/istio/pkg/revisions/default_watcher.go external/istio_new/pkg/revisions/default_watcher.go
+--- istio/pkg/revisions/default_watcher.go	2023-06-03 14:29:33.458875485 +0800
++++ istio_new/pkg/revisions/default_watcher.go	2023-06-03 14:29:27.410841215 +0800
+@@ -52,8 +53,12 @@
+ 		revision: revision,
+ 		mu:       sync.Mutex{},
+ 	}
+-	p.webhookInformer = client.KubeInformer().Admissionregistration().V1().MutatingWebhookConfigurations().Informer()
+-	p.webhookInformer.AddEventHandler(p.makeHandler())
++
++	// Commented by ingress
++	// Higress doesn't need to watch MutatingWebhookConfigurations, and it will break compatibility with K8s < v1.16.0.
++	// p.webhookInformer = client.KubeInformer().Admissionregistration().V1().MutatingWebhookConfigurations().Informer()
++	// p.webhookInformer.AddEventHandler(p.makeHandler())
++	// End commented by ingress
```

Comment:
Istiod has a env INJECT_ENABLED which determines the inject behavior.

Reply:
https://github.com/alibaba/higress/blob/80d6ecfddb0c349dfc453787d0b89d253530dba4/helm/core/templates/controller-deployment.yaml#L75-L77

Add env INJECT_ENABLED: false here to disable this webhook.

Reply:
Done. Please check it out.